src/css/style.css

```diff
@@ -1,3 +1,6 @@
+@import "services.css";
+
+
 * {
     box-sizing: border-box;
 }
```

Comment:
let's create a base.css and put all the tag selectors as well as this box sizing css in it, and put that import at the very top (above services.css).